pool/routes/routes.ml

```diff
@@ -113,12 +114,21 @@ module Contact = struct
         ; choose ~scope:(build_scope "sessions") sessions
         ]
       in
+      let locations =
+        let specific =
+          let open Location in
+          let files = [ choose ~scope:(add_key Field.File) [ get "" asset ] ] in
```

Comment:
I assume, we need to make sure, that not all assets are publicly available. According to the code, this "get asset" works for all files stored in our database.